examples/SimpleDrone/SimpleDrone.py

```diff
@@ -0,0 +1,244 @@
+
+        latlon_to_m = 111319.5   # converts lat/lon to meters
+        lat = x_ef / latlon_to_m + location.lat
+        lon = y_ef / latlon_to_m + location.lon
+        alt = z + location.alt
+
+
+        msg = self.vehicle.message_factory.set_position_target_global_int_encode(
+                                                    0,       # time_boot_ms (not used)
+                                                    0, 0,    # target system, target component
+                                                    mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT_INT, # frame
+                                                    0b0000111111111000, # type_mask (only speeds enabled)
+                                                    lat*1e7, # lat_int - X Position in WGS84 frame in 1e7 * meters
+                                                    lon*1e7, # lon_int - Y Position in WGS84 frame in 1e7 * meters
+                                                    alt, # alt - Altitude in meters in AMSL altitude, not WGS84 if absolute or relative, above terrain if GLOBAL_TERRAIN_ALT_INT
+                                                    0, # X velocity in NED frame in m/s
+                                                    0, # Y velocity in NED frame in m/s
+                                                    0, # Z velocity in NED frame in m/s
+                                                    0, 0, 0, # afx, afy, afz acceleration (not supported yet, ignored in GCS_Mavlink)
+                                                    0, 0)    # yaw, yaw_rate (not supported yet, ignored in GCS_Mavlink)
+
+        # send command to vehicle
+        self.vehicle.send_mavlink(msg)
+        self.vehicle.flush()
+        self.last_send_translate = time.time()
+        self.point(0)
+        print "translating..."
+        if wait_for_arrival:
+            while self.vehicle.mode == "GUIDED":
+                veh_loc = self.vehicle.location.global_relative_frame
+                diff_lat_m = (lat - veh_loc.lat) * latlon_to_m
+                diff_lon_m = (lon - veh_loc.lon) * latlon_to_m
+                diff_alt_m = alt - veh_loc.alt
+                dist_xyz = math.sqrt(diff_lat_m**2 + diff_lon_m**2 + diff_alt_m**2)
+                if dist_xyz < dist_thres:
+                    print "Arrived"
+                    return
+            print '\033[93m' + "SimpleDroneError: 'translate({0},{1},{2})' was interrupted. Vehicle was switched out of GUIDED mode".format(x,y,z) + '\033[0m'
+
+    # takeoff - takeoff to some altitude, needs armed status - params: meters
+    def takeoff(self, altitude_meters=3):
+        if self.vehicle.mode != 'GUIDED':
+            print '\033[91m' + "SimpleDroneError: 'takeoff({0})' was not executed. Vehicle was not in GUIDED mode".format(altitude_meters) + '\033[0m'
+            return
+
+        while not self.vehicle.armed:
+            print " Waiting for arming..."
+            time.sleep(1)
+
+        print "Taking off..."
```

Comment:
Fair enough. In the DK examples I tend to use an indent for this sort of pattern, so that the output looks a bit like:
```
Taking off ...
   Already airborne
```